src/libraries/tests.proj

```diff
@@ -379,6 +379,7 @@
     <!-- Run only a small randomly chosen set of passing test suites -->
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)*\tests\**\*.Tests.csproj" />
     <ProjectExclusions Remove="$(MSBuildThisFileDirectory)System.Collections\tests\System.Collections.Tests.csproj" />
+    <ProjectExclusions Remove="$(MSBuildThisFileDirectory)System.Reflection\tests\System.Reflection.Tests.csproj" />
```

Comment:
The hang when running the test seems to be Linux-specific, so following change should help us start getting coverage (and make sure we don't regress again) while the Linux hang is investigated:

```suggestion
    <ProjectExclusions Remove="$(MSBuildThisFileDirectory)System.Reflection\tests\System.Reflection.Tests.csproj" Condition="'$(TargetOS)' == 'windows'" />
```

Reply:
Thank!